src/ejabberd_c2s_state.erl

```diff
@@ -0,0 +1,12 @@
+-module(ejabberd_c2s_state).
```

Comment:
Before @bartekgorny changes, these functions were used from `mod_ping` as far as I remember. And if I'm correct, the purpose of the module was to have accessor to the c2s state so that there is no need to include header file with c2s state definition.

Reply:
It is true that since we decided to not pass c2s state to handlers, this modules becomes redundant.